osidb/api_views.py

```diff
@@ -517,6 +520,22 @@ class FlawReferenceView(
     permission_classes = [IsAuthenticatedOrReadOnly]
 
 
+@include_meta_attr_extend_schema_view
```

Comment:
CVSS does not seem to have any `meta_attr` attribute (and I think there is no reason it should have).